ui/src/dashboards/components/DashboardImportFromTemplateOverlay.tsx

```diff
@@ -75,7 +78,7 @@ class DashboardImportFromTemplateOverlay extends PureComponent<
           />
           <Overlay.Body>
             <div className="import-template-overlay">
-              <OrgTemplateFetcher orgName={this.orgName}>
+              <OrgTemplateFetcher orgID={orgID}>
```

Comment:
I think there's a get resources component that fetches templates and loads them into redux. maybe that would be better to sue for consistency